src/ResourceManager/Common/Commands.ResourceManager.Common/AzureRMCmdlet.cs

```diff
@@ -39,6 +40,14 @@ static AzureRMCmdlet()
             AzureSession.DataStore = new DiskDataStore();
         }
 
+        public AzureRMCmdlet()
+        {
+            AzureSession.ClientFactory.RemoveHandler(typeof(RPRegistrationDelegatingHandler));
```

Comment:
@jianghaolu this can be improved when we change the AddHandler to override existing handler.